addons/mail/static/src/rtc/call_action_list.xml

```diff
@@ -85,6 +85,17 @@
                             <i class="fa fa-compress fa-stack-1x" t-att-class="{ 'fa-lg': !isSmall }"/>
                         </div>
                     </button>
+                    <Dropdown t-if="moreOptions.length > 0" togglerClass="`${isSmall ? 'p-2' : 'p-3'} btn d-flex m-1 border-0 rounded-circle shadow-none opacity-100 opacity-75-hover`">
+                        <t t-set-slot="toggler">
+                            <div class="fa-stack">
+                                <i class="fa fa-ellipsis-h fa-stack-1x" t-att-class="{ 'fa-lg': !isSmall }"/>
+                            </div>
+                        </t>
+                        <DropdownItem t-foreach="moreOptions" t-as="option" onSelected="option.onSelected" t-key="option.label">
+                            <i t-if="option.icon" t-attf-class="me-2 {{option.icon}}"/>
```

Comment:
At the moment there is only one option, and that option will always have an icon, so I think this condition might be premature genericisation.

Reply:
Done